content/en/blog/_posts/2024-04-xx-structured-authz-beta.md

```diff
@@ -0,0 +1,324 @@
+        kubeConfigFile: /kube-system-authz-webhook.yaml
+        # matchConditions is a list of conditions that must be met for a request to be sent to this
+        # webhook. An empty list of matchConditions matches all requests.
+        # There are a maximum of 64 match conditions allowed.
+        #
+        # The exact matching logic is (in order):
+        #   1. If at least one matchCondition evaluates to FALSE, then the webhook is skipped.
+        #   2. If ALL matchConditions evaluate to TRUE, then the webhook is called.
+        #   3. If at least one matchCondition evaluates to an error (but none are FALSE):
+        #      - If failurePolicy=Deny, then the webhook rejects the request
+        #      - If failurePolicy=NoOpinion, then the error is ignored and the webhook is skipped
+      matchConditions:
+      # expression represents the expression which will be evaluated by CEL. Must evaluate to bool.
+      # CEL expressions have access to the contents of the SubjectAccessReview in v1 version.
+      # If version specified by subjectAccessReviewVersion in the request variable is v1beta1,
+      # the contents would be converted to the v1 version before evaluating the CEL expression.
+      #
+      # Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/
+      #
+      # only send resource requests to the webhook
+      - expression: has(request.resourceAttributes)
+      # only intercept requests to kube-system
+      - expression: request.resourceAttributes.namespace == 'kube-system'
+      # don't intercept requests from kube-system service accounts
+      - expression: !('system:serviceaccounts:kube-system' in request.user.groups)
+  - type: Node
+    name: node
+  - type: RBAC
+    name: rbac
+  - type: Webhook
+    name: in-cluster-authorizer
+    webhook:
+      authorizedTTL: 5m
+      unauthorizedTTL: 30s
+      timeout: 3s
+      subjectAccessReviewVersion: v1
+      failurePolicy: NoOpinion
+      connectionInfo:
+        type: InClusterConfig
+```
+
+The following configuration examples illustrate real world scenarios that need
+the ability to specify multiple webhooks with distinct settings, precedence
+order, and failure modes.
+
+### Protecting Installed CRDs
+Ensuring the availability of Custom Resource Definitions (CRDs) at cluster
+startup has been a key demand, One of the blockers for having a controller
+reconciling those CRDs is to have a protection mechanism for them, which can be
+achieved through multiple Authorization Webhooks. This was not possible before.
+Now with the Structured Authorization Configuration feature, administrators can
+specify multiple webhooks offering a solution where RBAC falls short, especially
+in denying permissions to 'non-system' users for certain CRDs.
+
+Assuming the following for this scenario:
+- The "protected" CRDs are installed in the kube-system namespace.
+- They can only be modified by users in the group
+  `system:serviceaccounts:kube-superuser`
```

Comment:
this means "any service account in the kube-superuser namespace"... was that intentional? if not, maybe swap this out for a more clearly fake example group (maybe a non-serviceaccounts one)?

Reply:
I took this from the KEP. it's using 
> They can only be modified by users in the group "system:serviceaccounts:kube-superuser"

how about `admin` or `crd-admin`

Reply:
yeah, either of those sgtm